web/sites/default/config/core.entity_view_display.media.iframe_embed.embedded_with_description.yml

```diff
@@ -0,0 +1,83 @@
+uuid: 062a9939-1732-4f03-850d-3303fca5f4b8
```

Comment:
To clean up the config, we can probably disable these "embedded_with_xxxxxxx" view modes on the Iframe and Video media types (unless there was a specific reason they were enabled in this PR). I believe they are only used on for document embeds.

![image](https://github.com/user-attachments/assets/a71b717a-3092-4adb-9e3e-281e008e2d72)

Reply:
Fixed